ACSS, PHP & AMSv2 public preview - version 1

gargankit-microsoft opened the pull request:
Please follow the link to find more details on the swagger existing in [azure-rest-api-specs-pr : private repo swagger](https://github.com/Azure/azure-rest-api-specs-pr/tree/RPSaaSMaster/specification/workloads/resource-manager/Microsoft.Workloads). That has already been reviewed by respective teams and is in private preview.

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [X] new service onboarding
      - [ ] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month. **June-July Timeframe**
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month. **As early as possible to support further rollouts and SDK Generation**
  4. If updating an existing version, please select the specific language SDKs and CLIs that must be refreshed after the swagger is published.
      - [x] SDK of .NET (need service team to ensure code readiness)
      - [x] SDK of Python
      - [x] SDK of Java
      - [x] SDK of Js
      - [x] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

Existing request for SDKs: https://github.com/Azure/sdk-release-request/projects/1164

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [x] Check this box if any of the following appy to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [x] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

gargankit-microsoft committed:
Initial changes for ACSS, PHP workloads under Microsoft.Workloads

gargankit-microsoft committed:
Updated documentation for AMS V2

gargankit-microsoft committed:
removed duplocate UserAssignedServiceIdentity object and added custom-words.txt

gargankit-microsoft committed:
updated custom-words.txt

gargankit-microsoft committed:
ran prettier check

gargankit-microsoft commented:
policy check is failing due to 

> The term [whitelist](http://policheck.azurewebsites.net/Pages/TermInfo.aspx?LCID=9&TermID=80411) was used in file 'custom-words.txt' and may not be appropriate to ship in all contexts.  When industry naming conventions or consistency make replacement of the term impractical, leave the term unchanged. When used to describe a list of people or things that have received approval or are considered to be in good favor, replace with "allow list". More specific terms such as "approved list" or "safe recipients list" can be used where appropriate.

Seems unrelated to our changes, as the flagged term is already there. The flag count of three seems to be related to three existing words in file

> whitelisted
> Whitelisting
> Whitelistings

gargankit-microsoft commented:
policy check is failing due to 

> The term [whitelist](http://policheck.azurewebsites.net/Pages/TermInfo.aspx?LCID=9&TermID=80411) was used in file 'custom-words.txt' and may not be appropriate to ship in all contexts.  When industry naming conventions or consistency make replacement of the term impractical, leave the term unchanged. When used to describe a list of people or things that have received approval or are considered to be in good favor, replace with "allow list". More specific terms such as "approved list" or "safe recipients list" can be used where appropriate.

Seems unrelated to our changes, as the flagged term is already there. The flag count of three seems to be related to three existing words in file

> whitelisted
> Whitelisting
> Whitelistings

gargankit-microsoft commented:
.NET SDK is failing for some time now for multiple PRs, hence our check is failing for that as well. this is unrelated to change and is caused due to script permissions, which is not in our control.
We are already in touch with owner of .NET SDK to get it generated offline.

gargankit-microsoft committed:
removed duplicate object defination

RamonArguelles committed:
Update Microsoft.MixedReality swagger to fix quality issues in s360 (#18835)

apattath committed:
Add Azure Communication Services Email data plane API (#19178)

* Add Azure Communication Services Email data plane API

* update SendMail.json to use 'CC' property

* Add suppression for linter rule DefinitionsPropertiesNamesCamelCase for CC proprety

kayousef committed:
Adding support for Auto LD (#19108)

heaths committed:
Merge Cognitive Services - Language pillar version 2022-05-15-preview to main (#19186)

* Updated May-1 preview to include changes from March-1 preview (#18580)

* Updated May-1 preview to include changes from March-1 preview

* Fixed a missed api version update in an example file

Co-authored-by: Abigail Hartman <Abigail.Hartman@microsoft.com>

* [Language API 2022-05-01-Preview]Conversation API Async - Conversational Summarization and Conversational PII (#17847)

* Adds base for updating Language from version preview/2022-02-01-preview to version 2022-04-01-preview

* Updates readme

* Updates API version in new specs and examples

* [Language API 2022-04-01-Preview] Feature conversation PII redaction and issue resolution summarization

* Add Issue Resolution Summary

* Address comments

* address pretty chek and spell check issues

* address file name mismatch issue

* typo

* Add IRD updates

* Add IRD Schema changes

* Response Files content update

* Address feedback

* Address PR comments

* Address comments

* Add the May API version

* Address comments

* Remove 04-01 and address comments

* Update specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/common.json

Co-authored-by: Heath Stewart <heaths@microsoft.com>

* Update specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/common.json

Co-authored-by: Heath Stewart <heaths@microsoft.com>

* Update specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/analyzeconversations.json

Co-authored-by: Heath Stewart <heaths@microsoft.com>

* Update specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/analyzeconversations.json

Co-authored-by: Heath Stewart <heaths@microsoft.com>

* Update specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/textanalytics.json

Co-authored-by: Heath Stewart <heaths@microsoft.com>

* Address feedback V-1.

* Address all comments expect model + enum descriptions

* Fx validations

* Missed files

* Conversation Item modality issue

* Update PII Task

* Address Spec validation errors

* Updates

* Fix PII issues

* Add readonly

* Update schema

* Updates

* Bring back Text Conversation Item

* Update example

* Space in Json

* Address comments

* Address one more comments

* Fix model validation

* Fix

* Update spec

* Address comments

* Move requestStatistics, prebuiltResult, error and warnings object to common

* Address circular dependency

Co-authored-by: Abigail Hartman <Abigail.Hahn@microsoft.com>
Co-authored-by: Heath Stewart <heaths@microsoft.com>

* Rename version from 2022-05-01-preview to 2022-05-15-preview (#18737)

* Update CLU swagger (#18739)

Co-authored-by: Chong Tang <chot@microsoft.com>

* Use azure-sdk-for-net-track2 for validation (#18755)

* Bidishac/update field description (#18787)

* Update the messaging on the parameter type

* Update the documentation of the redactionSource field

* Update specification/cognitiveservices/data-plane/Language/preview/2022-05-15-preview/analyzeconversations.json

Co-authored-by: Heath Stewart <heaths@microsoft.com>

Co-authored-by: Chong Tang <ct4ew@virginia.edu>
Co-authored-by: Heath Stewart <heaths@microsoft.com>

* Unify the runtime and authoring APIs (#18837)

* unifying single classification and multi classification to return array of class (#18902)

* Renaming in 2022-05-15-preview (#18929)

Co-authored-by: Chong Tang <chot@microsoft.com>

* Rename 'summary' in summaryAspects to 'issue' (#18941)

* Rename 'summary' in summaryAspects to 'issue'

* Merge issues

* Update model version value

* Merge 2022-05-01-GA authoring updates and "LUIS" string case update into 2022-05-15-preview branch (#18943)

* Merge 2022-05-01-GA files changes into 2022-05-15-preview branch

* Address Avocado issues and revert case update for Luis

* Add the missing error codes

* Fix avocado

* Avocado fix

* Add api-version parameter to cancel job endpoint (#18986)

* Add api-version parameter to cancel job endpoint

* Fix taskState and jobState

* Revert "Fix taskState and jobState"

This reverts commit 148dd2183249add89a090721db36a14c2b15d9bd.

* Update LUIS to Luis in model names (#19012)

* Copy QuestionAnswering definitions to common.json and refer them in analyzeconversations.json (#19031)

Co-authored-by: Chong Tang <chot@microsoft.com>

* Added fhir feature updates to 2022-05-15-preview (#19092)

Co-authored-by: Abigail Hartman <Abigail.Hartman@microsoft.com>

* Add cancel endpoint for conversations (#19025)

* Add cancel endpoint for conversations

* Update examples path

* Make the input for transcriptConversationItem optional

* Bidishac/update enum name (#19149)

* Update name to CreditCard

* Update entity categories for PII for build

* Keep SSN

* Revert "Keep SSN"

This reverts commit 77a92119ebd71cf5d5c7f201d402514756289ec2.

* Update readme.md

Co-authored-by: Abigail Hartman <Abigail.Hahn@microsoft.com>
Co-authored-by: Abigail Hartman <Abigail.Hartman@microsoft.com>
Co-authored-by: Bidisha Chakraborty <29803402+bidisha-c@users.noreply.github.com>
Co-authored-by: Chong Tang <ct4ew@virginia.edu>
Co-authored-by: Chong Tang <chot@microsoft.com>
Co-authored-by: Mohamed Abbas <moaba@microsoft.com>
Co-authored-by: Nourhan <92883092+nourhaanfarid@users.noreply.github.com>

heaths committed:
Cherry pick GA May APIs (#19188) (#19191)

* Cognitive Services Language GA API 2022-05-01 Swagger (#18402)

* adding Swagger for GA API

* remove extractive summerization changes

* run  .\Copy-AzSpecifications.ps1  -SourceBranch dev-cognitiveservices-Language-2022-03-01-preview

* removing extractive summerization from results

* removing analyze conversation APIs auto generated from March APIs

* single classification with array of class as multi classification

* update readme

* update the read me

* update read.me

* code review

* Use azure-sdk-for-net-track2 for validation (#18756)

* Add Language authoring 2022-05-01 APIs (#18694)

* Add Language authoring 2022-05-01 APIs

* Some build fixes

* Add polymerphism for CLU exported project

* Rename the api-version property in the import body

* Retrigger CI

* Update the hostTemplate and the operation Ids

* Use the common error objects

* Remove redundant definitions

* Update the stringIndexType

* Revert "Update the stringIndexType"

This reverts commit 455762d0443452299df363472bedbbd621ef7612.

* Fix import/export object in CT

* Rename actual to predicted

* Add the APIs examples

* Fix the examples validations

* Fix prettier

* Fix autorest issues

* Fix nextLink examples regression

* Retrigger CI

* Renaming the swagger files

* Apply PR feedback - part 1

* Add the documentation for the schemas

* Fix spelling errors

* Update the descriptions for the analyzetext-authoring

* Rename project-file-version to projectFileVersion

* Add CLU 2022-05-01 GA version (#18759)

* Add CLU 2022-05-01 GA version

* Remove role

* Update modality to inputModality

* Do not update property name to inputModality, only the enum name.

* Resolve comments

* Rename CustomConversation

Co-authored-by: Chong Tang <chot@microsoft.com>

* Update Language authoring and runtime APIs to follow the same convention (2022-05-01) (#18836)

* Update authoring and runtime APIs to follow the same convention

* Update the examples

* Remove other references to "named" in "named entity recognition"

* Remove more 'named' from the swagger

* Adding the missing 'required' attribute present in preview APIs and not in the GA version (#18946)

* Adding the missing 'required' attribute present in preview APIs and not in the GA version

* Updates

* Add api-version parameter to cancel job endpoint (#18985)

* Use PascalCase for enum values (#18840)

* rename LUIS to Luis

* use camel case for enum values

* Revert "use camel case for enum values"

This reverts commit 8db93fbb212cc73d33b77e17e0eadb4ed1204e6b.

Co-authored-by: Chong Tang <chot@microsoft.com>

* Copy QuestionAnswering definitions to common.json and refer them in analyzeconversations.json (#19030)

Co-authored-by: Chong Tang <chot@microsoft.com>

Co-authored-by: Heath Stewart <heaths@microsoft.com>
Co-authored-by: Mohamed Abbas <moaba@microsoft.com>
Co-authored-by: Chong Tang <ct4ew@virginia.edu>
Co-authored-by: Chong Tang <chot@microsoft.com>
Co-authored-by: Bidisha Chakraborty <29803402+bidisha-c@users.noreply.github.com>

Co-authored-by: Nourhan <92883092+nourhaanfarid@users.noreply.github.com>
Co-authored-by: Mohamed Abbas <moaba@microsoft.com>
Co-authored-by: Chong Tang <ct4ew@virginia.edu>
Co-authored-by: Chong Tang <chot@microsoft.com>
Co-authored-by: Bidisha Chakraborty <29803402+bidisha-c@users.noreply.github.com>

NetRock commented on specification/workloads/resource-manager/Microsoft.Workloads/preview/2021-12-01-preview/phpWorkloads.json:
```diff
@@ -0,0 +1,1362 @@
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Workloads/phpWorkloads/{phpWorkloadName}/wordpressInstances/default": {
```
Why "default"? is there any other wordpressInstance?

venu-sivanadham replied:
No, we want to restrict number of WordPress instances to 1. Hence using "default" as the only instance.

NetRock commented on specification/workloads/resource-manager/Microsoft.Workloads/preview/2021-12-01-preview/phpWorkloads.json:
```diff
@@ -0,0 +1,1362 @@
+          }
+        }
+      },
+      "put": {
```
This is t create/update default wordpressInstance?

gargankit-microsoft replied:
This is request contract we have in place to create that "default" instance

NetRock commented on specification/workloads/resource-manager/Microsoft.Workloads/preview/2021-12-01-preview/phpWorkloads.json:
```diff
@@ -0,0 +1,1362 @@
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Workloads/phpWorkloads": {
```
No CRUD on subscription level phpWorkloads?

gargankit-microsoft replied:
that would be below API

"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Workloads/phpWorkloads/{phpWorkloadName}"


Note: <b>Please follow the link to find more details on the swagger existing in [azure-rest-api-specs-pr: private repo swagger](https://github.com/Azure/azure-rest-api-specs-pr/tree/RPSaaSMaster/specification/workloads/resource-manager/Microsoft.Workloads). That has already been reviewed by respective teams and is in private preview right now.<b>

ArcturusZhang committed:
change operationIds (#18846)

jake-billings committed:
[Hub Generated] Publish private branch 'jakebillings/add-matching-resource-sync-rule' (#19034)

* add matchingResourceSyncRule to CustomLocationsFindTargetResourceGroup

* matching -> matched

* add missing x-ms-identifiers fields

* do a better job of fixing x-ms-identifiers identifier fields

zdelacerda-microsoft committed:
[Hub Generated] Review request for Microsoft.VirtualMachineImages to add version stable/2022-02-14 (#18896)

* Adds base for updating Microsoft.VirtualMachineImages from version stable/2021-10-01 to version 2022-02-14

* Updates readme

* Updates API version in new specs and examples

* adding 2022 changes

* fixed proxy resource path

* added object to ImageTemplate*

* added object to valide and prettier to examples

* added identifier

* removed format integer 32 for breaking change

* added back format integer 64, to be in compliance for lintdiff and breaking change

* adding back int32 after getting jeffrey richters approval

wwendyc committed:
[ProviderHub] S360 linter issue (#19067)

* x-ms-enum linter and prettier formatting

* formatting

* prettier

Co-authored-by: Wendy Chang <wendychang@gmail.com>

dw511214992 committed:
remove swagger-to-sdk except azure-sdk-for-net-track2 (#19000)

Juliehzl committed:
Fix examples in Microsoft.App (#19194)

* update

* fix

gargankit-microsoft commented on specification/workloads/resource-manager/Microsoft.Workloads/preview/2021-12-01-preview/SAPVirtualInstance.json:
```diff
@@ -0,0 +1,3324 @@
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Workloads/locations/{location}/sapVirtualInstanceMetadata/default/getSizingRecommendations": {
```
No, there are no other values. this is to have a post call with location-based routing.
sapVirtualInstanceMetadata is a ProxyOnly, LocationBased resource.

akashkeshari committed:
Introduce new preview api-version 2022-05-01-preview for Microsoft.Kubernetes (#19184)

* copied the base api-version(2021-10-01) to the new api-version(2022-05-01-preview) folder

* made changes for the new api-version (2022-05-01-preview)

* updated clusterName field in an example file

Co-authored-by: Akash Keshari <akkeshar@microsoft.com>

LinglingTong committed:
add xms-ids for postgresql (#17969)

gargankit-microsoft committed:
added UserAssignedServiceIdentity in monitors resource

ssss141414 committed:
[BotService] Update Slack property (#19105)

* Add blockchain to latest profile

* Add additional types

* update slack property

Co-authored-by: Mark Cowlishaw <markcowl@microsoft.com>

gargankit-microsoft committed:
added sku API signature

gargankit-microsoft committed:
updated readme with SKU defination

gargankit-microsoft committed:
updated documentation

krishna1s committed:
LoadTest Service: new preview api version  (#18169)

* adding data plane apis for load test service

* adding custom words

* Azure load test api 2022-03-01-preview

* New API version for fetching client metrics

* validation fixes

* typo fix

* resolving comments

* prettier run

* updated new examples

* renamed all files

* updated all secrets fields

* updates examples reference in spec

* updated pf criteria examples

* updated resource id field

* updated subnet fields

* updated all examples

* added host and schemes

* fix file type value

* renamed file

* updated all the examples

* removed validate file api ref and examples

* adding back all the params left out by oav tool

* resolve comments

* updated jmx filetype to be 0

* Improve descriptions

* updated nextlink

* resolving comments

* added filetype description

* update filetype enum

* Custom words

* Typo fixes

* Modify FileUrl description

* description "ISO 8601"

* matching regex for semantics

* move testId from params to inline - fix semantics

* moved api version def to inline

* added apiversion type string

* temp change dele app component

* semantics fix

* fixing testId and apiversion semantic

* added oneOf for schema

* testId query and path param changes

* appcomponent spec and example updates

* fix required type definition

* added readonly to testArtifacts

* removing required from resultUrl and logsUrl

* temp changes apiversion inline

* reverting temp changes

* fix lint validation

* adding required value in fileUrlList

* order testID param, refactor error, security scope

* file put api returns fileUrl model

* updated examples

* File api changes

* prettier fix

* adhering error model to rest api guidelines

* added next link in file url list

* updated example for nextlink fileUrlList

* added continuation token for get all files

* updated secret description

Co-authored-by: Krishna Chandak <krchanda@microsoft.com>
Co-authored-by: Venkat Ragavan <mail2venkat21@gmail.com>
Co-authored-by: Harshan B <harshan01@gmail.com>
Co-authored-by: harshan-b <101398067+harshan-b@users.noreply.github.com>

Noelin committed:
add readme.java.md and modify readme files (#18492)

* add readme.java.md and modify readme files

* update typescript sdk output-folder path

Alancere committed:
Fix examples for keyvault default tag: package-preview-2021-11 (#18962)

dw511214992 committed:
Add azure-sdk-for-net-track2 in private repo (#19214)

ChenglongLiu committed:
Fix lro in disable and enable cdn https (#19180)

* Fix lro in disable and enable cdn https

* Update the examples

* Update the examples

* Update the examples

* Update the examples

Co-authored-by: Chenglong Liu <chengll@microsoft.com>

liurunliang-bot committed:
Add PRLabels to CODEOWNERS (#19123)

* Add PRLabels to CODEOWNERS

Init - Add PRLabel to CODEOWNERS

* Update CODEOWNERS

Co-authored-by: Mike Kistler <mikekistler@microsoft.com>

* Update CODEOWNERS

Co-authored-by: Mike Kistler <mikekistler@microsoft.com>

* Add fabricbot.json

Co-authored-by: Ruoxuan Wang <52271048+ruowan@users.noreply.github.com>
Co-authored-by: Mike Kistler <mikekistler@microsoft.com>

colawwj committed:
machinelearningservice-typescript.md-update (#19213)

cohen-uriel-ms committed:
Adding Microsoft.Sql AdvancedThreatProtectionSettings APIs on v5 tag for 2021-11-01-preview (#19216)

erjms committed:
[Hub Generated] Review request for Microsoft.MachineLearningServices to add version stable/2022-05-01 (#18645)

* Adds base for updating Microsoft.MachineLearning from version stable/2019-10-01 to version 2022-05-01

* Updates readme

* Updates API version in new specs and examples

* add 2022-05-01 mfe.json

* add 2022-02-01-preview mls.json

* add workspaceFeatures.json from 2022-02-01-preview

* add example files

* update readme

* Update mfe spec

- Update discriminator values from PascalCase to snake_case
- Remove job schedules
- Add MLTable

* Add ScheduleBase types to machineLearningServices.json

* Updated custom-words.txt

* Update examples

* Update readme

* add workspace connection v2 changes

* Update readme

* Update readme

* fix example validation error

* Update mfe swagger

* Updated examples

* Moved 2022-05-01 specs from machinelearning to machinelearningservices

* Update readme

* port 2022-02-01-preview mfe.json Schedule to machineLearningServices.json

* update example w/ provisioningState

* Rename workspace connection related auth type

* description change to try to trigger pipelines

* Revert "description change to try to trigger pipelines"

This reverts commit 899cb59290814f98b56bc777cf4f369eb2ed78c8.

* update missed common-types v2 -> v3

* Remove unnecessary Credentials properties

* fix propertyBag schema

* add x-ms-secret

* remove checkNameAvailability api

* remove traffic property from PartialOnlineEndpoint

* prettier

* Revert "port 2022-02-01-preview mfe.json Schedule to machineLearningServices.json"

This reverts commit 4268210b92105c3c7ba89b2d45d7bcd0c56c2864.

* add v1LegacyMode to workpsace properties.

* update description and add default value.

* update description

* [MLC] label sslConfiguration and virtualMachineImage as nullable

* reduce PATCH to bare minimum

* update examples w/ new spec

* [MLC] Mark as null-able for properties Getting API

* TEST enum as x-nullable

* Revert "TEST enum as x-nullable"

This reverts commit 616291c334fa825ac07a8b589797478699c7f6b6.

* [MLC] Mark as null-able for properties of List API

* remove x-nullable from properties with defaults

* description update

* enum ordering

* fix XmsEnumValidation error for MLC

* change x-ms-enum name of SSLConfiguration

* update descriptions

* change x-ms-client-name 'Details' to 'Properties'

* removed x-ms-client-name 'Data' suffix

* reorder SecretsType enum

Co-authored-by: Teddy <thtodoro@microsoft.com>
Co-authored-by: zhaomuzhi <zhaoweixiong521@gmail.com>
Co-authored-by: Xi Jin <jinxi@microsoft.com>
Co-authored-by: ZhidaLiu <86350902+ZhidaLiu@users.noreply.github.com>
Co-authored-by: Teddy Todorov <forteddyt@gmail.com>
Co-authored-by: Bingchen Li <bingchenli@microsoft.com>

audunn committed:
[NetAppFiles] Anf 19190 update for 2022 01 01 (#18516)

* Add api version dir

* Apply 2022-01-01 Changes

* Remove Default from throughputMibps

* add list x-ms-identifiers

* Bump api version in examples

* pretty, spelling

* fix operations lists example

* fix lro exampes

* add systemData to 2021-08-01

* apply list identifyer added in another PR to previous version

* Encyprtionkeysource enum

* review comment fixes

* Fixes quotaRule

* Referenc common types

* common types

* more common

* Remove regionInfo endpoint

* Revert NetAppAccount using common types due to linDiff validation

* Revert NetAppAccount using common types due to linDiff validation

* Revert NetAppAccount using common types due to linDiff validation

* Remove regionInfo endpoint2

* merge

* fixes EncryptionKeySource

* fixes EncryptionKeySource.

* x-ms-identifiers

* sibling rivalry

* add regionInfo

* add regionInfo.json

* etag

* etag

* etag 2021-08-01

* remove regionInfo

* VolumeQuotaRule response code

* proxy resource from v3

colawwj committed:
update-for-typescript.md (#19192)

Commit:
[Relay] Adding missing success response code to PUT calls on Private Endpoints (#19156)

rchlkm committed:
[Hub Generated] Review request for Microsoft.SqlVirtualMachine to add version preview/2021-11-01-preview (#18838)

* Add Everyday enum to autoPatching.dayOfWeek

* rename duplicate x-ms-enum

* rename PatchingDayOfWeek

* rename days of week variants

* rename AutoBackupDaysOfWeek

khaledaboasbe committed:
[Hub Generated] Review request for Microsoft.AlertsManagement to add version stable/2019-03-01 (#18656)

* Add comment to Change state parameters

* fix error

* fix error

* fix

* fix

* fix error

* fix validations

* add example for operation list command

* fix example

* fix example

* fix

* fix

* delete json file

* add origin parameter to operation defenition

* fix

* fix

* fix

* delete unneccesary files

* revert packege-lock.json file changes

Co-authored-by: Khaled Abo Asbe <khaboasb@microsoft.com>

moabba committed:
Fix Language authoring policheck issues (#19223)

* Fix Language authoring policheck issues

* Replace the API response with the latest value

* Rename `Persina (Farsi) to `Persian`

* Revert "Rename `Persina (Farsi) to `Persian`"

This reverts commit 5a1feff2a7b01f25b63829bf29ccf0fd9f66cb52.

* Rename `Persian (Farsi)` to `Persian`

* Remove the Bengali example

vutran01 committed:
update etag and kind type (#18652)

* initial expected changes

* remove additionalProperties from etag

* rename property name

kseager committed:
update description for todos for Feb (#19230)

Co-authored-by: Kayla Seager <kaseager@microsoft.com>

gargankit-microsoft committed:
reverted MSI changes for monitors, would be taken up separately

gargankit-microsoft commented:
merged from wrong branch, will raise another one with clear diffs

gargankit-microsoft committed:
Merge remote-tracking branch 'origin/main' into ACSS_PHP_AMSv2_PublicPreview_Version1

gargankit-microsoft commented:
moved to https://github.com/Azure/azure-rest-api-specs/pull/19232 as this was having issues in running checks
